In Python 3.7, SSL certificate validation is enabled by default. This is a problem
for Splunk as we ship with a self-signed certificate. This patch merely sets it
back to the previous behavior. You can read more here:

https://www.python.org/dev/peps/pep-0476/#opting-out

--- ssl.py	2019-03-29 11:55:36.200312854 -0700
+++ build/linux_x86_64/lib/python3.7/ssl.py	2018-12-14 11:36:43.000000000 -0800
@@ -621,7 +621,8 @@
     return context
 
 # Used by http.client if no context is explicitly passed.
-_create_default_https_context = _create_unverified_context
+_create_default_https_context = create_default_context
+
 
 # Backwards compatibility alias, even though it's not a public name.
 _create_stdlib_context = _create_unverified_context
